main/guides/js-programming/notifiers.md

```diff
@@ -152,7 +152,7 @@ Zoe provides updates on the state of seats within a contract. The updates
 from Zoe indicate changes to the allocation of a seat and seats exiting. 
 These are available from `E(userSeat).getNotifier()` and `zcfSeat.getNotifier()`, 
 which provide long-lived notifier objects associated with a particular 
-seat. `zcfSeats` are available within contracts while `userSeats` are accessible 
+seat. `zcfSeat`s are available within contracts while `userSeat`s are accessible 
```

Comment:
These still don't seem to align with https://agoric.com/documentation/zoe/api/zoe-contract-facet.html , perhaps the following would be better?
```suggestion
seat. `ZCFSeat`s are available within contracts while `UserSeat`s are accessible 
```

Reply:
@erights ?

Reply:
The upper case clearly refers to the types, which stands for all the instances of that kind. The lower case is how you'd refer to an instance, which in context clearly also stands for all instances of that kind. I'm ok either way, so whichever you prefer. The main thing is consistency throughout this and related documents.

Reply:
Yes, that doc clearly establishes that we lean towards the upper case type-name form. I like that.